requirements.txt

```diff
@@ -1,3 +1,5 @@
-numpy
-scipy
+# install package from setup.py
+# NOTE: -e . for editable mode
+-e .
```

Comment:
The previous `setup.py` opened `requirements.txt` and used its contents as `install_requirements=...`. This now moved to `setup.cfg` and I would argue that we do not need `requirements.txt` anymore -- at least we do not use it currently. 

Is it related to future conda packaging?

Reply:
no, I tought this would be used to setup the environment for testing that. So if you develop the package, you would write a new test, and this would require to setup a specific test_environment. This could/should contain more specific version informations used for testing (this would then be used for building the tests in a CI).

Reply:
I see and agree. I propose to define minimal versions the dependencies (numpy + scipy) in setup.cfg. It would be reasonable to base them on the version in some popular OS, like [ubuntu18.05 LTS](https://packages.ubuntu.com/bionic/python-numpy) (so numpy>=1.13.3). Then, we would run two set of tests in our CI:
1. just `pip install -e .` with any numpy+scipy version from the current CI container that fulfils our requrements (e.g. numpy 1.19.1)
2. specific dependency versions with the _minimal_ version, so numpy==1.13.3

For clarity we should then rename the file to "minimal_requirements.txt" or something.